docs/docsite/rst/roadmap/COLLECTIONS_9.rst

```diff
@@ -40,7 +40,7 @@ Release schedule
 
 .. [2] After this date only changes blocking a release are accepted. Accepted changes require creating a new release candidate and may slip the final release date.
 
-.. [3] Collections will be updated to a new version only if a blocker is approved. Collection owners should discuss any blockers at a community IRC meeting (before this freeze) to decide whether to bump the version of the collection for a fix. See the `Community IRC meeting agenda <https://github.com/ansible/community/issues/539>`_.
+.. [3] Collections will be updated to a new version only if a blocker is approved. Collection owners should discuss any blockers at a Community working group meeting (before this freeze) to decide whether to bump the version of the collection for a fix. See the `Community IRC meeting agenda <https://github.com/ansible/community/issues/645>`_.
 
 .. note::
 
```

Comment:
@samccann I think this can also be changed as https://forum.ansible.com/t/vote-ends-on-2023-12-05-stop-using-the-gh-com-ansible-community-meeting-agenda-for-the-community-wg-meeting/2243/11.
This statement as a whole feels outdated. I think we should ask them to create a community topic https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md#creating-a-topic

@felixfontein @gotmax23 @mariolenz ^ ideas

Reply:
:+1: indeed, creating a `community-wg` forum post seems like the best way forward.

Reply:
Sorry for the delay in getting back to this. @Andersson007 @felixfontein can you take another look pls?

Reply:
Please see @mariolenz [comment](https://github.com/ansible/ansible-documentation/pull/913#discussion_r1462350623) as it suggests we shouldn't be editing this at all for the Ansible 9 roadmap.